ui/page/dexclient/create_wallet_modal.go

```diff
@@ -90,10 +90,12 @@ func (md *createWalletModal) Dismiss() {
 
 func (md *createWalletModal) OnDismiss() {
 	md.ctxCancel()
+	md.sourceAccountSelector.UnsubscribeTxNotifications()
 }
 
 func (md *createWalletModal) OnResume() {
 	md.ctx, md.ctxCancel = context.WithCancel(context.TODO())
+	md.sourceAccountSelector.SubscribeTxNotifications()
 	md.sourceAccountSelector.ListenForTxNotifications(md.ctx)
```

Comment:
Explains a previous comment about automatically subscribing the listener when `ListenForTxNotifications` is called. Don't need to be a separate method.